New package: CompressedBeliefMDPs v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: CompressedBeliefMDPs
- Repository: https://github.com/JuliaPOMDP/CompressedBeliefMDPs.jl
- Created by: @FlyingWorkshop
- Version: v1.0.0
- Commit: 49a36512fb66f7595a603e4246730e52b55fa36a
- Reviewed by: @FlyingWorkshop
- Reference: https://github.com/JuliaPOMDP/CompressedBeliefMDPs.jl/issues/9#issuecomment-2053839923
- Description: Compressed belief-state MDPs in Julia compatible with POMDPs.jl
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835980fca22515f812e889513aab36d1ed19529ff3b171ee76fda4c52c59cefcc488f866e1c08944fe00e3974994cee921946a7dd3a7a6fa290d9ea865d070cc52615c4b9791855eadd5c3d87b951bbcff55cf13e49b7881cf574176f2e5d43e7c9724e1e9e2e82689cb04aa76ff21d550d83bec6ca893a2ada7a692aee078100ebeed1eda72bc2e9324db6b058a0e3e9422a -->

FlyingWorkshop commented:
We've already solicited feedback [here](https://discourse.julialang.org/t/pre-ann-compressedbeliefmdps-jl-solve-large-pomdps-in-pomdps-jl/112625) :D

JuliaRegistrator committed:
New package: CompressedBeliefMDPs v1.0.0

UUID: 0a809e47-b8eb-4578-b4e8-4c2c5f9f833c
Repo: https://github.com/JuliaPOMDP/CompressedBeliefMDPs.jl.git
Tree: a396a52acf611577e255e68d3946234eb3b7b6d2

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
